ieproxy_windows.go

```diff
@@ -76,7 +76,25 @@ func parseRegedit(regedit regeditValues) ProxyConf {
 }
 
 func readRegedit() (values regeditValues, err error) {
-	k, err := registry.OpenKey(registry.CURRENT_USER, `Software\Microsoft\Windows\CurrentVersion\Internet Settings`, registry.QUERY_VALUE)
+	// Check config per user or per machine
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `Software\Policies\Microsoft\Windows\CurrentVersion\Internet Settings`, registry.QUERY_VALUE)
+	if err != nil {
+		return
+	}
+	defer k.Close()
+	proxySettingsPerUser, _, err := k.GetIntegerValue("ProxySettingsPerUser")
+	if err != nil && err != registry.ErrNotExist {
+		return
+	}
+	var hkey registry.Key
+	if err == nil && proxySettingsPerUser == 0 {
```

Comment:
You did an error check on line 86, why did you repeat the error check?

Furthermore, this means that if the error is anything other than registry.ErrNotExist, we default to CURRENT_USER. This seems like incredibly odd and potentially buggy behavior to me. Is this behavior intended?